device.c

```diff
@@ -354,13 +346,13 @@ static int vcam_s_parm(struct file *file,
     return 0;
 }
 
-static int vcam_enum_framesizes(struct file *filp,
+static int vcam_enum_framesizes(struct file *file,
```

Comment:
Don't use `file` when you refer to a pointer to `struct file`. Instead, you can use `f`.

Reply:
But in every other `v4l2_ioctl_ops` callback function we use `file` instead.
Should I change every other parameter names together?

Reply:
The build-in callback function `[vb2_ioctl](https://elixir.bootlin.com/linux/latest/source/drivers/media/common/videobuf2/videobuf2-v4l2.c#L988)` also use the name `file`.

Reply:
Affirmative! Let's use `file`.